uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Helper/Menu/Parser.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+use CRM_HRCore_Menu_Item as MenuItem;
+
+class CRM_HRCore_Helper_Menu_Parser {
```

Comment:
What do you think of renaming this to something like `CRM_HRCore_Menu_CiviAdapter`, as this is actually adapting the format from CiviHR to CiviCRM?

By the book, the Adapter pattern is used to adapt the interface of a class to another class, which is not exactly what were doing here, since what we're doing here is adapting things from one system to another, I think the concept can still be used.